Assets/Scripts/EnviornmentalFeatures/BloodGeyser.cs

```diff
@@ -4,8 +4,18 @@
 
 public class BloodGeyser : MonoBehaviour
 {
+    [Header("Audio")]
+    [SerializeField] private float bloodGeyserAudioVolume = 1f;
+    [SerializeField] private float bloodGeyserAudioDistanceRange = 250f;
+
     float acceleration;
     float maxSpeed;
+
+    private void Start()
+    {
+        AudioManager.GetInstance().PlayAudioAtLocation("BloodGeyser_SFX", transform.position, bloodGeyserAudioVolume, true, bloodGeyserAudioDistanceRange);
```

Comment:
The audio doesn't stop when the geyser disappears

Reply:
Fixed!